lib/modules/platform/gitea/index.ts

```diff
@@ -200,6 +201,12 @@ const platform: Platform = {
       botUserID = user.id;
       botUserName = user.username;
       defaults.version = await helper.getVersion({ token });
+      if (defaults.version?.includes('+gitea-')) {
+        defaults.version = defaults.version.substring(
+          defaults.version.indexOf('+gitea-') + 7,
+        );
```

Comment:
wouldn't this be an empty string all the time?

Reply:
no, see tests. I've discussed this with the forgejo maintainers.